agent/07_agent_add_node.sh

```diff
@@ -33,14 +31,36 @@ if [ -f $OCP_DIR/add-node/node.iso ]; then
   rm -f $OCP_DIR/add-node/node.iso
 fi
 
-oc adm node-image create --dir "$OCP_DIR/add-node/" --registry-config "${PULL_SECRET_FILE}" --loglevel=2
+tmpaddnodedir=$(mktemp --tmpdir -d "addnode--XXXXXXXXXX")
+_tmpfiles="$_tmpfiles $tmpaddnodedir"
+cp -rf $OCP_DIR/add-node/* $tmpaddnodedir
+
```

Comment:
I'm not sure about the benefits of creating (yet another) temp folder in this case?
Why not simply reuse the same $OCP_DIR/add-node/?

Reply:
The $OCP_DIR/add-nodes/nodes-config.yaml file is consumed after executing "oc adm node-image create". Using a temp directory allows us to run "make agent_add_node" multiple times.

Reply:
Sorry @rwsu I missed this comment. How is consumed?

Reply:
Ok, nodes-config.yaml wasn't actually being consumed but rather .openshift_install_state.json was blocking subsequent executions of "oc adm node-image create". I will remove the temp folder and add code to remove .openshift_install_state.json.